### PR TITLE
fix: Remove multiple durations for seated events

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

This PR disables multiple durations for seated events.

Fixes # (issue)

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- In an event type enable multiple durations
- Now enable seats
- Navigate back to the multiple durations. It should be back to a single duration and the toggle should be disabled

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/pages/event-types/[type]/index.tsx [hunk 1]
```diff
@@ -278,6 +279,8 @@ const EventTypePage = (props: EventTypeSetupProps) => {
               .filter((slug) => slug !== eventType.slug) ?? [],
         },
       })),
+      multipleDurationEnabled: eventType.metadata?.multipleDuration || false,
+      seatsPerTimeSlotEnabled: eventType.seatsPerTimeSlot,
```
Comment (joeauyeung) [inline apps/web/pages/event-types/[type]/index.tsx:283]: Enable this default value as you had to navigate to the "Advanced" tab for this value to appear

File: apps/web/components/eventtype/EventAdvancedTab.tsx [hunk 2]
```diff
@@ -338,6 +338,7 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
                   toggleGuests(false);
                   formMethods.setValue("requiresConfirmation", false);
                   setRequiresConfirmation(false);
+                  formMethods.setValue("metadata.multipleDuration", undefined);
```
Comment (joeauyeung) [inline apps/web/components/eventtype/EventAdvancedTab.tsx:341]: Remove all multiple durations when seats are enabled

**Review by leog — DISMISSED** [review]
LGTM, any test to update for this?

**Review by keithwillcode — APPROVED** [review]

**Review by leog — APPROVED** [review]
LGTM

**Pull request merged**